djangocms_pageadmin/admin.py

```diff
@@ -70,9 +93,14 @@ def state(self, obj):
     state.short_description = _("state")
 
     def url(self, obj):
-        path = obj.page.get_path(obj.language)
-        if path is not None:
-            url = obj.page.get_absolute_url(obj.language)
+        path = obj._path
+        url = None
+        with override(obj.language):
+            if obj.page.is_home:
+                url = reverse("pages-root")
+            if path:
```

Comment:
Is it intended that even if obj.page.is_home is true that a reverse will be used anyway? OR is path always false if the obj is the homepage?